packages/core/test/Serialization/TransactionBody/ProposalProcedure/TreasuryWithdrawalsAction.test.ts

```diff
@@ -1,13 +1,17 @@
 /* eslint-disable sonarjs/no-duplicate-string */
 import * as Cardano from '../../../../src/Cardano';
+import { Hash28ByteBase16 } from '@cardano-sdk/crypto';
 import { HexBlob } from '@cardano-sdk/util';
 import { TreasuryWithdrawalsAction } from '../../../../src/Serialization';
 
 // Test data used in the following tests was generated with the cardano-serialization-lib
-const cbor = HexBlob('8202a1581de1cb0ec2692497b458e46812c8a5bfa2931d1a2d965a99893828ec810f01');
+const cbor = HexBlob(
+  '8302a1581de1cb0ec2692497b458e46812c8a5bfa2931d1a2d965a99893828ec810f01581c8293d319ef5b3ac72366dd28006bd315b715f7e7cfcbd3004129b80d'
```

Comment:
@mirceahasegan same as comment above, just confirm this has been verified with at least one more sources (CSL or ledger)

Reply:
Verified with @emurgo/cardano-serialization-lib-nodejs@12.0.0-alpha.16